2018-11-17 16:01:32,296 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:01:33,292 [open_context_base.py] on_error:307: Connect error: conn_id=1; err=WSAECONNREFUSED;
2018-11-17 16:01:33,296 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-17 16:01:41,297 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:01:42,299 [open_context_base.py] on_error:307: Connect error: conn_id=2; err=WSAECONNREFUSED;
2018-11-17 16:01:42,309 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-17 16:01:50,312 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:01:51,312 [open_context_base.py] on_error:307: Connect error: conn_id=3; err=WSAECONNREFUSED;
2018-11-17 16:01:51,318 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-17 16:01:59,331 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:02:00,332 [open_context_base.py] on_error:307: Connect error: conn_id=4; err=WSAECONNREFUSED;
2018-11-17 16:02:00,338 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-17 16:02:08,339 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:02:09,335 [open_context_base.py] on_error:307: Connect error: conn_id=5; err=WSAECONNREFUSED;
2018-11-17 16:02:09,345 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-17 16:02:17,345 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:02:17,347 [open_context_base.py] on_connected:289: Connected : conn_id=6; 
2018-11-17 16:02:18,118 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=6; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 2965874871121607559, 'conn_key': '80FB753D7BE828BE', 'keep_alive_interval': 10};
2018-11-17 16:02:18,127 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:02:18,137 [open_context_base.py] on_connected:289: Connected : conn_id=7; 
2018-11-17 16:02:18,139 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=7; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 16224474373136275332, 'conn_key': '8857373E7BEA28D6', 'keep_alive_interval': 10};
2018-11-17 16:02:18,151 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-17 16:02:18,152 [open_context_base.py] on_connected:289: Connected : conn_id=8; 
2018-11-17 16:02:18,154 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=8; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 5920240624736076036, 'conn_key': '05013B3E7BEC2849', 'keep_alive_interval': 10};
